webrtc-stats.html

```diff
@@ -2029,6 +2030,23 @@ <h2>
                   regardless if the RID RTP header extension has been negotiated.
                 </p>
               </dd>
+              <dt>
+                <dfn>encodingIndex</dfn> of type <span class="idlMemberType">
+                unsigned long</span>
+              </dt>
+              <dd>
+                <p>
+                  MUST NOT [= map/exist =] for audio.
+                  Only [= map/exist =]s if this <a>RTP stream</a> belongs to an
+                  RTP sender that has been configured for simulcast, in other
+                  words the {{RTCRtpSender}} has multiple
+                  {{RTCRtpSendParameters/encodings}} and
+                  {{RTCOutboundRtpStreamStats}} objects. The {{encodingIndex}}
+                  is the index of the encoding that represents this <a>RTP
+                  stream</a>. MUST NOT be [= map/exist | present =] for
+                  singlecast senders.
```

Comment:
Maybe add zero for simulcast?

Reply:
0 for singlecast*, yes will do

Reply:
Updated to say that this is the index without any talk about simulcast or singlecast, the implication is that this always has a value regardless